ID.me WebVerify SDK/IDmeWebVerify.m

```diff
@@ -51,118 +53,151 @@ + (IDmeWebVerify *)sharedInstance {
 - (id)init{
     self = [super init];
     if (self) {
+        self.keychainData = [[IDmeWebVerifyKeychainData alloc] init];
         [self clearWebViewCacheAndCookies];
     }
     
     return self;
 }
 
++ (void)initializeWithClientID:(NSString *)clientID redirectURI:(NSString *)redirectURI {
+    [[IDmeWebVerify sharedInstance] setClientID:clientID];
```

Comment:
do we want to support changing the clientID after the initialize method was called? I think these properties should be readonly and here we should call a private constructor which pass the id and url